meterfs/test_meterfs_miscellaneous.c

```diff
@@ -100,15 +100,16 @@ TEST(meterfs_miscellaneous, resize_bigger)
 TEST(meterfs_miscellaneous, multi_lookup)
 {
 	file_info_t info = { 2, 2000, 20, 0 };
-	char name[] = "file0";
+	const char *name = "file0";
 	int i;
 
 	TEST_ASSERT_EQUAL(0, file_allocate(name, info.sectors, info.filesz, info.recordsz));
 
-	for (i = 0; i < 5; ++i)
-		TEST_ASSERT_GREATER_OR_EQUAL(0, file_lookup(name));
+	(void)snprintf(common.buffMsg, sizeof(common.buffMsg), "/%s", name);
```

Comment:
If you fix `lookup_rel` leading `/` will not be needed

Reply:
Do we need these (void) casting?

Reply:
@jsarzy 
MISRA Rule 17.7 The value returned by a function having non-void return type shall be used

This rule is a required one

P.S. I guess this `/` is not needed now.

Reply:
it fails on PC without it